core/src/com/mygdx/game/client/modules/GameStateVerifierModule.java

```diff
@@ -0,0 +1,22 @@
+package com.mygdx.game.client.modules;
+
+import com.mygdx.game.client.service.GameStateVerifier;
+import com.mygdx.game.client.service.LocalVerifier;
+import com.mygdx.game.client.service.TurnService;
+import dagger.Module;
+import dagger.Provides;
+import lombok.extern.java.Log;
+
+import javax.inject.Singleton;
+
+@Module
+@Log
+public class GameStateVerifierModule {
```

Comment:
Ten GameStateVerifier chyba jednak do usunięcia, bo w innym PR przekształciłem to na wysyłanie poleceń do serwera.

Reply:
Ok, to zobaczyłem trochę do kodu znowu i z tego co widzę to jeśli by to miało być używane to ta implementacja, która teraz jest, jest bardziej implementacją lokalną serwera. (Ale raczej nie szedłbym w ten model GameStateVerifiera, tylko w te synchronizatory z #40). 

Lokalna implementacja serwera byłaby teraz po stronie aplikacji, bo nie mamy jeszcze komunikacji z serwerem. (Z którą mam trochę problem, bo albo nie mogę się zdecydować jak ją zrobić jeśli chodzi o zamysł [UDP vs TCP, biblioteki javowe vs jakieś netty reactor], albo mam problemy z implementacją (w przypadku tego netty reactor). )

Reply:
Sam schemat działania tej klasy chyba nie jest do końca zły.